browser/help/topics/v4-hts.md

```diff
@@ -66,11 +66,549 @@ The above example will retrieve the entire frequency struct for each variant. To
+- `age_distribution`: Callset-wide age histogram calculated on release samples.
+  - `bin_edges`: Bin edges for the age histogram.
+  - `bin_freq`: Bin frequencies for the age histogram. This is the number of records found in each bin.
+  - `n_smaller`: Count of age values falling below lowest histogram bin edge.
+  - `n_larger`: Count of age values falling above highest histogram bin edge.
+- `downsamplings`: Dictionary keyed by dataset with values corresponding to available downsampled sample counts.
+- `filtering_model`: The variant filtering model used and its specific cutoffs.
+  - `filter_name`: Variant filtering model name used in the 'filters' row annotation, indicating the variant was filtered by this model during variant QC.
+  - `score_name`: Annotation name of the score used for variant filtering.
+  - `snv_cutoff`: SNV filtering cutoff information.
+    - `bin`: Filtering percentile cutoff for SNVs.
+    - `min_score`: Minimum score at SNV filtering percentile cutoff.
+  - `indel_cutoff`: Indel filtering cutoff information.
+    - `bin`: Filtering percentile cutoff for indels.
+    - `min_score`: Minimum score at indel filtering percentile cutoff.
+  - `snv_training_variables`: Variant annotations used as features in the SNV filtering model.
+  - `indel_training_variables`: Variant annotations used as features in the indel filtering model.
+- `inbreeding_coeff_cutoff`: Inbreeding Coefficient threshold used to hard filter variants.
+- `interval_qc_parameters`: Thresholds used to determine whether an exome calling interval was high coverage and passed interval QC.
+  - `per_platform`: Whether these thresholds were stratified per platform.
+  - `all_platforms`: Whether these thresholds were applied uniformly across platforms, as long as a platform had a sample size above `min_platform_size`.
+  - `high_qual_cutoffs`: Interval QC thresholds per chromosomal region:
+    - `autosome_par`: Contains annotation used to filter high coverage intervals on autosomes and in pseudoautosomal regions and threshold for annotation. Interval was considered high quality only if this annotation was over the specified threshold value.
+    - `x_non_par`: Contains annotation used to filter high coverage intervals in non-pseudoautosomal regions of chromosome X and threshold for annotation. Interval was considered high quality only if this annotation was over the specified threshold value.
+    - `y_non_par`: Contains annotation used to filter high coverage intervals in non-pseudoautosomal regions of chromosome Y and threshold for annotation. Interval was considered high quality only if this annotation was over the specified threshold value.
+  - `min_platform_size`: Sample size required for a platform to be considered.
```

Comment:
I noticed it's like this in release HT: 
```commandline
        all_platforms: bool, 
        high_qual_cutoffs: dict<str, array<tuple (
            str, 
            str, 
            float64
        )>>, 
        min_platform_size: int32
```
Under high_qual_cutoffs, there isn't any field name.

Reply:
do you mean that there aren't direct names under `high_qual_cutoffs`? that is true, but if you eval that dict:
```
>>> ht = hl.read_table('gs://gnomad/release/4.1/ht/exomes/gnomad.exomes.v4.1.sites.ht')
>>> hl.eval(ht.interval_qc_parameters)
Struct(per_platform=False, all_platforms=True, high_qual_cutoffs={'autosome_par': [('fraction_over_20x', 'all', 0.85)], 'x_non_par': [('fraction_over_20x', 'XX', 0.85), ('fraction_over_10x', 'XY', 0.85)], 'y_non_par': [('fraction_over_10x', 'XY', 0.85)]}, min_platform_size=100)
```
so I added this description here to make this field easier to understand